pkg/apis/networkoperator/v1/networkconfig_types.go

```diff
@@ -179,6 +182,9 @@ const (
 
 	// NetworkTypeRaw
 	NetworkTypeRaw NetworkType = "Raw"
+
+	// NetworkTypeOpenShiftSRIOV
+	NetworkTypeOpenshiftSRIOV NetworkType = "OpenshiftSRIOV"
```

Comment:
OpenShift b/c Dan Winship is watching :-)

Reply:
Just need to fix the capitalization.

Reply:
Done